openlibrary/templates/books/custom_carousel.html

```diff
@@ -40,7 +40,7 @@
   $else:
     $ img_attr = 'src'
 
-  <div class="book carousel__item">
+  <div class="book opacity-hide opacity-transition carousel__item">
```

Comment:
Could you use BEM notation for these classes? E.g. carousel__item--opacity..


http://getbem.com/introduction/

Reply:
Sure!